spec/models/event_spec.rb

```diff
@@ -89,6 +137,56 @@
     end
   end
 
+  describe '#scheduled?' do
+    it { expect(event.scheduled?).to eq false }
+    it 'returns true if the event is scheduled' do
+      event.room = create(:room)
+      event.start_time = conference.start_date.to_time
+      expect(event.scheduled?).to eq true
+    end
+  end
+
+  describe '#registered_text' do
```

Comment:
This should be a [helper spec](https://relishapp.com/rspec/rspec-rails/docs/helper-specs/helper-spec)